packs/october.js

```diff
@@ -0,0 +1,39 @@
+const icon = `data:image/svg+xml,%3Csvg width='310' height='310' xmlns='http://www.w3.org/2000/svg' preserveAspectRatio='xMidYMid meet'%3E%3Cg%3E%3Ctitle%3Eoctober%3C/title%3E%3Crect fill='none' id='canvas_background' height='602' width='802' y='-1' x='-1'/%3E%3C/g%3E%3Cg%3E%3Ctitle%3ELayer 1%3C/title%3E%3Cg id='svg_1' fill='white' transform='translate(0,310) scale(0.10000000149011612,-0.10000000149011612) '%3E%3Cpath fill='%23de6c26' id='svg_2' d='m1350,3031c-142,-44 -349,-218 -499,-420c-293,-396 -455,-869 -437,-1281c8,-202 49,-355 144,-545c136,-274 408,-551 655,-669c141,-67 134,-69 141,28c3,44 10,324 16,621c27,1373 44,1760 82,1913c6,23 14,42 19,42c12,0 36,-91 49,-183c5,-43 10,-177 10,-298c0,-120 3,-219 7,-219c3,0 57,50 119,110c69,68 120,110 133,110c18,0 19,-3 10,-27c-12,-31 -79,-132 -191,-285l-78,-108l1,-312c1,-172 4,-374 8,-449l6,-137l110,126c140,160 351,371 395,396l33,19l-6,-32c-20,-98 -95,-207 -374,-543l-163,-196l0,-311c0,-183 4,-311 9,-311c28,0 193,64 268,105c138,73 238,150 383,295c157,157 244,274 334,452c205,400 210,803 16,1190c-178,356 -546,721 -878,869c-117,53 -246,73 -322,50z'/%3E%3C/g%3E%3C/g%3E%3C/svg%3E`;
+
+const UIStrings = {
+  /** Additional description of a Lighthouse audit that tells the user how they can improve performance by removing unused CSS, in the context of the October CMS CMS platform. This is displayed after a user expands the section to see more. No character length limits. Links in (parenthesis) become link texts to additional documentation. */
```

Comment:
I see a lot of "CMS CMS"–should that just be CMS?

These strings are just for our translators so it doesn't really matter, it may be confusing though.

Reply:
Sort of, the problem is that the name is itself "October CMS".

"The October CMS CMS platform" could definitely be turned into just "October CMS" though.

Reply:
Removed all "CMS CMS" and switched to "CMS"